examples/cgroupstats_dump.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2017-present, Outlyer
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import os
+import sys
+from gnlpy.cgroupstats import CgroupstatsClient
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(
+        description='Show cgroup load information. Run as root.')
+    parser.add_argument('path', type=str, help='Path to cgroup cpu hierarchy')
+
+    args = parser.parse_args(argv[1:])
+    c = CgroupstatsClient()
+    fd = os.open(args.path, os.O_RDONLY)
```

Comment:
this also seems closer to the kernel api: http://lxr.free-electrons.com/source/tools/accounting/getdelays.c#L427